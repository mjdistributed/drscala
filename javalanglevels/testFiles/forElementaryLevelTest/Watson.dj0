class Watson extends AFriend {
  String getName() {
    return "Watson";
  }
}
