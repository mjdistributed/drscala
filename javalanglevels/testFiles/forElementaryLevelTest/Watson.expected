// Language Level Converter line number map: dj*->java. Entries: 5
//     1->3         2->4         3->5         4->6         5->7    
class Watson extends AFriend {
  public String getName() {
    return "Watson";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public Watson() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      Watson cast = ((Watson) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}
